src/engine/strat_engine/pool.rs

```diff
@@ -71,19 +73,20 @@ impl StratPool {
         {
             let mut data_devs = Vec::from_iter(bds.values());
 
-            meta_dev = try!(LinearDev::new(&format!("stratis_{}_meta", name),
-                                           dm,
-                                           &vec![data_devs.remove(0)]));
+            let meta_blockdev = data_devs.remove(0);
+            meta_dev =
+                try!(LinearDev::new(&format!("stratis_{}_meta", name), dm, &vec![meta_blockdev]));
+            try!(wipe_sectors(&try!(meta_dev.path()), &Sectors(0), &DATA_BLOCK_SIZE));
```

Comment:
You probably need a comment here, to inform posterity why you had to wipe these sectors.